packages/client/src/generation/utils/buildDirname.ts

```diff
@@ -34,12 +35,23 @@ function buildDirnameFind(defaultRelativeOutdir: string, runtimePath: string) {
 
   return `
 const { findSync } = require('${runtimePath}')
+const fs = require('fs')
 
-const dirname = findSync(process.cwd(), [
+// some frameworks or bundlers replace or totally remove __dirname
+const hasDirname = typeof __dirname !== 'undefined' && __dirname !== '/'
+
+// will work in most cases, ie. if the client has not been bundled
+const regularDirname = hasDirname && fs.existsSync(path.join(__dirname, 'schema.prisma')) && __dirname
+
+// if the client has been bundled, we need to look for the folders
+const foundDirname = !regularDirname && findSync(process.cwd(), [
     ${defaultRelativeOutdir ? `${JSON.stringify(defaultRelativeOutdir)},` : ''}
     ${serverlessRelativeOutdir ? `${JSON.stringify(serverlessRelativeOutdir)},` : ''}
-], ['d'], ['d'], 1)[0] || __dirname`
+], ['d'], ['d'], 1)[0]
+
+const dirname = regularDirname || foundDirname || __dirname`
```

Comment:
How can someone easily debug this, for example looking at logs?
(Should we somehow have debug logs for each const?)

Reply:
I can easily add a debug log for `dirname` so that one can see the path that was found.

Reply:
Ideally, we would have any notion of paths here and bundle the schema on generate. But that's another topic to explore.

Reply:
I added this into to the client debug logs.